cmd/explaintest/t/select.test

```diff
@@ -89,6 +89,7 @@ select !(1 + 2);
 # - +
 select + - 1, --1, +-+-+1, + "123";
 select --------------------1, ++++++++++++++++++++1;
+select +(+(1)), (-+1), ((+1)), +1.23, +null, +true, +false;
```

Comment:
How about add `+1e23`, `+1E23`, since we have discussed this situation in that issue.

Reply:
OK, I will add it.